_includes/activities.html

```diff
@@ -79,3 +79,4 @@
   </footer>
 </div>
 </div>
+<script src="/assets/js/search.js"></script>
```

Comment:
Added to es, fr and pt and used Google Translate for localised strings. However, I've included an English fallback in case they are forgotten.